clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -83,12 +84,14 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+@SuppressWarnings("classfanoutcomplexity")
 public class ConsumerNetworkThreadTest {
 
     private ConsumerTestBuilder testBuilder;
     private Time time;
     private ConsumerMetadata metadata;
     private NetworkClientDelegate networkClient;
+    private BlockingQueue<BackgroundEvent> backgroundEventsQueue;
```

Comment:
seems we don't use this anymore right? And probably don't need the supresswarnings either?

Reply:
right, I miss that